src/Pages/HomePage/HouseIntro/HouseIntro.jsx

```diff
@@ -98,7 +98,12 @@ export default () => {
                         <p
                           className={`font-semibold text-md lg:text-xl relative after:scale-x-0 after:h-0.5 after:bottom-0 after:left-0 after:absolute after:origin-bottom-left after:transform after:ease-in-out after:duration-500 cursor-pointer w-full after:w-full group-hover:after:scale-x-100 group-hover:after:origin-bottom-left after:${house.themecss.textGradient} text-gray-600 dark:${house.themecss.textColor}`}
                         >
-                          {house.name}
+                          {house.name.split('\n').map((line, index) => (
```

Comment:
since we are getting only name property from housedata.house we can directly write elements like 
House of <br /> {house.name}

this is better and easy approach

Reply:
Also if you use React you need to import it first, i think you did not tested it locally, other wise you should have got error in loading page,
but here we will not need it, still remember this